### ISSUE TITLE
[Feature] [MicroBot] Add COVID Info Microbot

### ISSUE THREAD

**Issue by satyammittal** [scalability4all/voice-enabled-chatbot#51]
**Is your feature request related to a problem? Please describe.**
Taking current situation into consideration, It would be good to have a bot for COVID. it should tell current active/ total cases based on your location. 

**Describe the solution you'd like**

"Am I safe from COVID":
based on location, it tells whether you are in green/yellow/red zone

"Can I go outside?"
You are under lockdown. Please stay home.

"How many cases are there?"
There are 289 total cases, 50 recovered and 0 deaths.

**Describe alternatives you've considered**
API parsing and a simple tool.

**Additional context**

**Comment by rishier827** [comment]
Can i get assigned

**Comment by KC2016** [comment]
Can we make suggestions as well?

**Comment by satyammittal** [comment]
@KC2016 Yes sure, you can also start working on it. As it would be highly useful in this situation, i encourage everyone to put extra hands.
